src/pyqasm/visitor.py

```diff
@@ -47,6 +47,11 @@
 logger = logging.getLogger(__name__)
 logger.propagate = False
 
+class LoopControlSignal(Exception):
```

Comment:
Can you please define this in the [`src/pyqasm/exceptions.py`](https://github.com/qBraid/pyqasm/blob/main/src/pyqasm/exceptions.py) file?